### PR TITLE
add a little more auditing

### PR DESCRIPTION
**Pull request by akostadinov**
This is the part of #3342 that is ready to merge.

### CODE REVIEW

File: app/models/backend_api.rb [hunk 1]
```diff
@@ -5,6 +5,8 @@ class BackendApi < ApplicationRecord
   include SystemName
   include ProxyConfigAffectingChanges::ModelExtension
 
+  audited :allow_mass_assignment => true
```
Comment (jlledom) [inline app/models/backend_api.rb:8]: what is this for?

File: app/services/audit_log_service.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class AuditLogService
+  include Callable
+
+  def self.call(message)
+    Rails.logger.info "[AUDIT]: #{message}"
+  end
+end
```
Comment (jlledom) [inline app/services/audit_log_service.rb:7]: No new line at the end of the file!

File: app/models/backend_api.rb (hunk 1 above)
Comment (akostadinov) [reply app/models/backend_api.rb:8]: One audit for all changes, not one per change. We use it everywhere.

File: app/services/audit_log_service.rb (hunk 2 above)
Comment (akostadinov) [reply app/services/audit_log_service.rb]: It's a good practice. I don't know why codeclimate doesn't complain about it. Fixed.

**Review by jlledom — APPROVED** [review]

**Pull request merged**